spec/support/shared_context/rake_helper.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+# https://robots.thoughtbot.com/test-rake-tasks-like-a-boss
+require "rake"
+
+shared_context "rake" do
```

Comment:
That rename seems like a clever idea, but it's lying on the content of the file. I think the ideal would be to filter out `spec/support/**.*` altogether.